drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/dmn/KogitoScenarioSimulationBuilder.java

```diff
@@ -125,7 +125,7 @@ private void convertScenarioSimulationModel(final ScenarioSimulationModel toConv
         JsUtils.setNameOnWrapped(scesim, makeJSINameForSCESIM());
         JsUtils.setValueOnWrapped(scesim, jsiScenarioSimulationModelType);
         SCESIMMarshallCallback scesimMarshallCallback = getSCESIMMarshallCallback(callback);
-        SCESIMMainJs.marshall(scesim, "", scesimMarshallCallback);
+        SCESIMMainJs.marshall(scesim, null, scesimMarshallCallback);
```

Comment:
@danielzhe According to the code, it seems to me that `dynamicNamespace` parameter was not used in SCESIM even before your changes.and both `""` and `SCESIM` were unused placeholder (and the same is for the unmarsharllaing process). Fine for me, I just ask you to manual test SCESIM Kogito version, just to be sure. If you already did that, even better.